src/utilities/getLocale.ts

```diff
@@ -0,0 +1,14 @@
+import { allLocales } from "../generated/locale-codes";
+
+type LangKey = "tr" | "en";
+
+export const getLocale = (): LangKey => {
```

Comment:
`configureLocalization` exposes a getLocale method. We can export that from localization.ts and use that instead. Issue detail might have been misleading sorry about that.